internal/profile/_static/kibana_config_8x.yml

```
@@ -55,7 +56,7 @@ xpack.fleet.outputs:
   - id: fleet-default-output
     name: default
     type: elasticsearch
-    hosts: [ https://elasticsearch:9200 ]
+    hosts: [https://elasticsearch:9200]
```

```suggestion
    hosts: ["https://elasticsearch:9200"]
```

Mainly to be consistent.